example_vst2_windows_test.go

```diff
@@ -0,0 +1,88 @@
+	},
+})
+
+func ExampleWindows() {
+	// VST2 files have OS-specific extensions.
+	pluginPath := "_testdata/TAL-Reverb.dll"
```

Comment:
I'm not 100% sure, but I believe that this plugin doesn't support double precision processing. 

Please make the changes from comments to check this hypothesis.

Reply:
The plugin `TAL-Reverb.dll` does handle both double and float buffers - but only if two channels are provided.
Thus currently the example `example_vst2_windows_test.go` fails on my windows machine.

I tried `pipelined/vst2`with a bunch of vst plugins on my machine. All of the free TAL vst2 plugins are failing if used with the example data.
But some other free (and commercial) plugins are running without the null pointer exception (with double and float) - at least the 64bit versions.

To merge this PR i suggest either:
1. use the `TAL-Reverb.dll` and add a second channel to the testdata and change the `// Output:` to match it.
or
2. add a free plugin that works with one channel and change the `// Output:` to match it. On my machine [https://www.looperman.com/free-music-software/detail/nick-crow-7170-lead](https://www.looperman.com/free-music-software/detail/nick-crow-7170-lead) does work.

Further suggestions:
Double precision could be used again. I´m not a travis guy but it would be nice to execute the tests on darwin and on windows.

Reply:
Thank you for checking this with other plugins, very valuable insight! 

> use the TAL-Reverb.dll and add a second channel to the testdata and change the // Output: to match it.

Let's stick to this plan, so we know that this plugin causes the error. I created #16 to track progress there. 

> Double precision could be used again. 

Yes, we can fallback to double precision now.

> I´m not a travis guy but it would be nice to execute the tests on darwin and on windows.

Agree, I created #17 to work on it.

Let me know if you want to fallback to DoubleProcessing here, otherwise we can land.